perf-tests/clusterloader2/pkg/chaos/nodes.go

```diff
@@ -111,7 +111,7 @@ func (k *NodeKiller) pickNodes() ([]v1.Node, error) {
 		return nil, err
 	}
 
-	prometheusPods, err := client.ListPodsWithOptions(k.client, monitoringNamespace, metav1.ListOptions{
+	prometheusPods, err := client.ListPodsWithOptions(k.client, util.GetTenant(), monitoringNamespace, metav1.ListOptions{
```

Comment:
Does this still work for the default case (scale-up) where no tenant has been specified?

Reply:
yes.

If when no env var of SCALEOUT_TEST_TENANT is specified, the function will return "system" tenant. 

This is also verfied via a test run.